packages/docs/package.json

```diff
@@ -23,6 +23,7 @@
   "scripts": {
     "test:packages": "cd ../web && yarn test:unit:generate && cd ../mobile && yarn test:unit:generate",
     "build": "build-storybook -o dist",
+    "build:clean": "rm -rf dist",
```

Comment:
Due to our needs to some files inside the dist folder, we cannot use this lifecycle. A solution here would be making a `build:local` and a `build:local:clean` lifecycles,  point it to another folder and make it ignored on the `.gitignore`

Reply:
I'll stop pointing the dev server to the `dist` folder, that should be enough